config/routes.rb

```diff
@@ -1,3 +1,21 @@
 Rails.application.routes.draw do
-  # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
+  get '/users', to: 'users#index'
+  get '/friend_requests', to: 'friendships#friend_requests'
+  get '/friends', to: 'friendships#friends'
+  
+  resources :posts
```

Comment:
posts, likes, and friendship belong to the user and as such should be nested within the user route.
since only the post can be commented, comments should be nested within the post route.